include/shellstrings.h

```diff
@@ -41,7 +53,18 @@ void shell_intro();
  * Returns:
  *  - None.
  */
-void shell_save();
+void shell_save(bool *color);
+
+/*
+ * shell_edit_success - file editing success message
+ *
+ * Parameters:
+ *  - None
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_edit_success(bool *color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!